internal/process/datacenter.go

```diff
@@ -97,12 +97,12 @@ func createDatacenterSamples(config *config.Config) {
 		checkError(config.Logrus, ms.SetMetric("datastore.totalUsedGiB", (totalDatastoreCapacity-totalDatastoreFreeSpace)/(1<<30), metric.GAUGE))
 
 		checkError(config.Logrus, ms.SetMetric("overallStatus", string(dc.Datacenter.OverallStatus), metric.ATTRIBUTE))
-		checkError(config.Logrus, ms.SetMetric("datastores", len(dc.Datastores), metric.GAUGE))
+		checkError(config.Logrus, ms.SetMetric("datastoreCount", len(dc.Datastores), metric.GAUGE))
 		checkError(config.Logrus, ms.SetMetric("hostCount", len(dc.Hosts), metric.GAUGE))
-		checkError(config.Logrus, ms.SetMetric("vmCount", len(dc.VirtualMachines), metric.GAUGE))
-		checkError(config.Logrus, ms.SetMetric("networks", len(dc.Networks), metric.GAUGE))
-		checkError(config.Logrus, ms.SetMetric("resourcePools", countResourcePools, metric.GAUGE))
-		checkError(config.Logrus, ms.SetMetric("clusters", len(dc.Clusters), metric.GAUGE))
+		checkError(config.Logrus, ms.SetMetric("virtualMachineCount", len(dc.VirtualMachines), metric.GAUGE))
+		checkError(config.Logrus, ms.SetMetric("networkCount", len(dc.Networks), metric.GAUGE))
+		checkError(config.Logrus, ms.SetMetric("resourcePoolCount", countResourcePools, metric.GAUGE))
+		checkError(config.Logrus, ms.SetMetric("clusterCount", len(dc.Clusters), metric.GAUGE))
```

Comment:
This is a braking change for queries and alerts i think.
If we carry on with this i would suggest to add it in a different PR also.

Reply:
I agree, It would require as well a change in documentation/shim file, I would not change that

Reply:
Ok, i'll revert it.
We need to make sure next time to have a more consistent naming. Some use count, some use plural for the same type of "thing"